### DISCUSSION TITLE
How can I change Font Family in Dev Container?

### DISCUSSION THREAD

**Discussion by hhimanshu** [devcontainers/spec#346] [Q&A] [answered]
Hello here

I want to enable two specific use cases - programming, and writing blogs. For programming, I want to keep my default font, but when enabling a dev container that is meant for blog writing, I want to change the font family. Is it possible to do so?

Can I switch to a different font-family when starting a dev container?

Thank you

**Comment by bamurtaugh** [comment, marked as answer]
Thanks for sharing this. Have you tried placing the font you want in the `vscode.settings` property: https://containers.dev/supporting#visual-studio-code?

Example:

```
"settings": { 
    "editor.fontFamily": "'Courier New', monospace"
  }
```

**Reply by hhimanshu** [reply]
Thank you @bamurtaugh

**Reply by hhimanshu** [reply]
Your videos on YouTube are great, btw @bamurtaugh

**Reply by bamurtaugh** [reply]
No problem! And thank you so much for watching and for the kind feedback 😄
